svo (install the zip, not me).xml

```diff
@@ -54937,7 +54937,7 @@ end
 -- downloads the system &amp; updates the system version saved
 function svo.downloadnewsystem(newversion)
   svo.downloadedsystem = downloadfolder..string.format(&quot;%s svo.zip&quot;, svo.me.name)
-  local downloadurl = [[http://vadisystems.com/svo-r/r/]]..svo.me.name..[[%20svo.zip]]
+  local downloadurl = [[http://svof.pathurs.com/]]..svo.me.class..[[%20svo.zip]]
```

Comment:
I will try to find all references now and change to for example: jester.svof.v8.zip